### ISSUE TITLE
Exit with code 1 due to network error: ContentNotFoundError

### ISSUE THREAD

**Issue by sumankr1** [SebastiaanKlippert/go-wkhtmltopdf#69]
I'm trying to run some javascript code inside the `script` tag. Here is the console error output.
```
Loading pages (1/6)
Counting pages (2/6)                                               
Resolving links (4/6)                                                       
Loading headers and footers (5/6)                                           
Printing pages (6/6)
Done                                                                      
Exit with code 1 due to network error: ContentNotFoundError
```

As soon as I remove JS from the HTML file, the PDF is generated properly.

Here is the JS snippet:

```js
 <script>
      function includeHTML() {
        var z, i, elmnt, file, xhttp;
        /*loop through a collection of all HTML elements:*/
        z = document.getElementsByTagName("*");
        for (i = 0; i < z.length; i++) {
          elmnt = z[i];
          /*search for elements with a certain atrribute:*/
          file = elmnt.getAttribute("w3-include-html");
          if (file) {
            /*make an HTTP request using the attribute value as the file name:*/
            xhttp = new XMLHttpRequest();
            xhttp.onreadystatechange = function () {
              if (this.readyState == 4) {
                if (this.status == 200) {
                  elmnt.innerHTML = this.responseText;
                }
                if (this.status == 404) {
                  elmnt.innerHTML = "Page not found.";
                }
                /*remove the attribute, and call this function once more:*/
                elmnt.removeAttribute("w3-include-html");
                includeHTML();
              }
            };
            xhttp.open("GET", file, true);
            xhttp.send();
            /*exit the function:*/
            return;
          }
        }
      }
      includeHTML();
    </script>
```

**Comment by sumankr1** [comment]
Thanks! There was a problem from my side.

**Comment by TaylorRayHoward** [comment]
Just a note for anyone in the future, you might be able to ignore this error and the pdf still be generated correctly. 

I was able to get the bytes even though this error was cropping up for me. In my use-case, I don't particularly care if specific content can't load and `LoadErrorHandling("ignore")` [wasn't working for me](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2051).  I was able to check the content of the error and ignore it.

```go
if !strings.Contains(fmt.Sprint(err), "ContentNotFoundError") {
  return err
}
```
